lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -47,6 +47,10 @@ def command
     task.application_name
   end
 
+  def comment
+    task.comment
+  end
+  
```

Comment:
Ok, I thought it was really trivial, but if you want I'm happy to take it out and resubmit.